frontend/src/components/common/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,52 @@
+import { Icon, IconifyIcon } from '@iconify/react';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export interface ActionButtonProps {
+  /** Label is a description of the action.
+   *
+   * Internally it is passed into t() i18n translation.
```

Comment:
This works for existing translations but will not work for ones that need to be parsed into a catalog.
So I'd actually not pass this into a translation function but just describe how to translate strings.